testgrid/config.yaml

```diff
@@ -7052,6 +7057,11 @@ dashboards:
 
 - name: presubmits-kubernetes-nonblocking
   dashboard_tab:
+  - name: pull-kubernetes-e2e-kind
+    test_group_name: pull-kubernetes-e2e-kind
+    base_options: width=10
+    alert_options:
+      alert_mail_to_addresses: bentheelder+alerts@google.com
   - name: pull-kubernetes-cross
```

Comment:
pretty sure this should be in blocking 🤔

Reply:
opened https://github.com/kubernetes/test-infra/issues/11453